tensorrt_llm/_torch/speculative/mtp.py

```
@@ -1236,123 +1250,134 @@ def prepare_position_ids_and_last_tokens(position_ids, attn_metadata):
+                            hidden_states_gathered.view(
+                                -1, hidden_states_gathered.shape[-1]),
+                            (0, 0, 0, pad_len),
+                            mode="constant",
+                            value=0)
+                    elif pad_len == 0:
+                        padded_hidden_states = hidden_states_gathered.view(
+                            -1, hidden_states_gathered.shape[-1])
+                    else:
+                        raise ValueError(
+                            f"In MTPEagleWorker.forward(), token_count < max_num_requests, which is not supported"
+                        )
```

Ruff (0.14.10)

Avoid specifying long messages outside the exception class

(TRY003)